### PR TITLE
[WIP] Arryan/ttc api

### PR DESCRIPTION
**Pull request by Arryan**
- add api client for nextbus
docs: https://www.nextbus.com/xmlFeedDocs/NextBusXMLFeed.pdf

### CODE REVIEW

File: v2/src/Interfaces.ts [hunk 1]
```diff
@@ -21,3 +21,8 @@ export interface CarouselVehicle {
   terminal: string,
   delay: number
 }
+
+export interface Point {
```
Comment (dellisd) [inline v2/src/Interfaces.ts:25]: I think we should use GeoJSON conventions for representing points. They're represented as `[longitude, latitude]`.

Comment (Arryan) [reply v2/src/Interfaces.ts:25]: Fair enough, i can transform the response to match that format

File: v2/src/Api.tsx [hunk 2]
```diff
@@ -67,7 +32,7 @@ export interface ApiDirection {
 
 export interface ApiPath {
   /* [[lon, lat]] */
-  point: [[number, number]] 
+  point: [[number, number]]
```
Comment (dellisd) [inline v2/src/Api.tsx:35]: Is this intended to be an array with a single point element?

Comment (Arryan) [reply v2/src/Api.tsx:35]: it's supposed to be an array of points, so like `[[lon1,la1], [lon2,lat2], ...]`

Comment (dellisd) [reply v2/src/Api.tsx:35]: I think that would have to be `[number, number][]`

Comment (Arryan) [reply v2/src/Api.tsx:35]: Ok thanks I wasn't sure

**Pull request merged**